ui/components/AutomationDetail.tsx

```diff
@@ -57,6 +119,29 @@ function AutomationDetail({
     conditions,
     sourceRef,
   } = automation;
+  const reconciledObjectsAutomation: ReconciledObjectsAutomation = {
+    name,
+    namespace,
+    clusterName,
+    type: Kind[type],
+    suspended,
+    conditions,
+    source: sourceRef,
+  };
+  const { path } = useRouteMatch();
+  const { data, isLoading, error } = useGetInventory(
+    type,
+    name,
+    clusterName,
+    namespace,
+    false,
+    {
+      retry: false,
+      refetchInterval: (data) =>
```

Comment:
There is an `enabled` flag that you can use to turn the hook on an off.

```suggestion
      enabled: (data) =>
```

Reply:
Taking in consideration that user might navigate from/to different tabs and refresh the page or uses a link to specific tab other than Details Or navigate back and forward when Details is not the active tab  ... Than enabled won't work as expected